library/alloc/src/alloc.rs

```diff
@@ -361,13 +365,22 @@ extern "Rust" {
 /// [`set_alloc_error_hook`]: ../../std/alloc/fn.set_alloc_error_hook.html
 /// [`take_alloc_error_hook`]: ../../std/alloc/fn.take_alloc_error_hook.html
 #[stable(feature = "global_alloc", since = "1.28.0")]
+#[rustc_const_unstable(feature = "const_box", issue = "none")]
```

Comment:
This may need to be changed to an another feature flag.

Reply:
const_alloc_error I guess?